.github/workflows/python.yml

```diff
@@ -82,7 +82,7 @@ jobs:
           set -ex
           python3 -m venv .venv
           source .venv/bin/activate
-          pip install eppo-server-sdk --find-links dist --force-reinstall
+          pip install eppo-server-sdk --no-index --find-links dist --force-reinstall
```

Comment:
Without `--no-index`, the CI was [randomly](https://github.com/Eppo-exp/rust-sdk/actions/runs/11388672269) selecting either local package or one published on pypi. See https://github.com/Eppo-exp/rust-sdk/pull/50/commits/36f083b2b5d78e69a0167c6fa8916ba03892573c

Reply:
kind of wild default behavior 😅